hoomd/jit/patch.py

```diff
@@ -226,6 +245,53 @@ def compile_user(self, array_size_iso, array_size_union, code, clang_exec, fn=No
 
         return llvm_ir
 
+    def wrap_gpu_code(self, code, union = False):
+        R'''Helper function to compile the provided code into a device function
+
+        Args:
+            code (str): C++ code to compile
+            clang_exec (str): The Clang executable to use
+            fn (str): If provided, the code will be written to a file.
+            array_size_iso (int): Size of array with adjustable elements for the isotropic part. (added in version 2.8)
+            array_size_union (int): Size of array with adjustable elements for unions of shapes. (added in version 2.8)
```

Comment:
This docstring is from the `user` function and needs to be updated.

Reply:
done